### PR TITLE
Fix(logger): Ignore docker health check requests

### PR DESCRIPTION
**Pull request by licy183**
### Component/Part
logger

### Description
This patch disables logging of HTTP requests containing health-check UA from the local IP address. These logs can take up a lot of disk space but are of little use.

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### Related Issue(s)
#1933

### CODE REVIEW

**Comment by mrdrogdrog** [comment]
The request is always sent from 127.0.0.1. So why have you implemented all these edge cases?

**Comment by licy183** [comment]
> The request is always sent from 127.0.0.1. So why have you implemented all these edge cases?

If the checker's client IP address is always 127.0.0.1, it is indeed unnecessary to implement these test cases. Should I submit a commit to delete these test cases and use 127.0.0.1 only?

File: app.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ if (!config.useSSL && config.protocolUseSSL) {
 
 // logger
 app.use(morgan('combined', {
+  skip: (req, _) => req.headers['user-agent'] === 'hedgedoc-container-healthcheck/1.0' && req.ip === '127.0.0.1',
```
Comment (mrdrogdrog) [inline app.js:68]: Please extract the check into a separate function. Like `IsRequestFromContainerHealthCheck()` or something.

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (licy183) [reply app.js]: OK, I'll fix it.

**Review by mrdrogdrog — APPROVED** [review]

**Review by davidmehren — APPROVED** [review]
Thanks!

**Review by DerMolly — APPROVED** [review]

**Review by InnayTool — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]
